File: Exercicios-Treinamento/C++/Sequencial13.cpp
#include <iostream>
using namespace std;
//Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:
//Para homens: (72.7*h) - 58
//Para mulheres: (62.1*h) - 44.7
int main (){
    float h;
    cin >> h;
    float masc;
    masc = (72.7*h)-58;
    float fem;
    fem = (62.1*h)-44.7;

    cout << masc << "Homens"<<endl;
    cout << fem << "Mulheres"<<endl;

}

File: Exercicios-Treinamento/Python/Sequencial13.py
"""
 Tendo como dado de entrada a altura (h) de uma pessoa, 
 construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:
    Para homens: (72.7*h) - 58
    Para mulheres: (62.1*h) - 44.7
"""
h = float(input("Digite a sua altura :  "))
homem = round( ((72.7*h) - 58),2)
mulher = round(((62.1*h) - 44.7),2)
print("Peso ideal para Homen = ",homem," Peso ideal mulher = ", mulher)
